roles/ceph-validate/tasks/check_system.yml

```diff
@@ -46,11 +46,11 @@
     - ceph_repository == 'uca'
     - ansible_distribution != 'Ubuntu'
 
-- name: "fail on unsupported openSUSE distribution (only 15.x supported)"
+- name: "fail on unsupported SUSE/openSUSE distribution (only 15.x supported)"
   fail:
-    msg: "Distribution not supported: {{ ansible_distribution }}"
+    msg: "Distribution not supported: {{ ansible_distribution }} {{ ansible_distribution_major_version }}"
   when:
-    - ansible_distribution == 'openSUSE Leap'
+    - ansible_distribution == 'openSUSE Leap' or ansible_distribution == 'SUSE'
```

Comment:
I'm not very familiar with openSUSE/SUSE releases.
just to be sure: regardless we are deploying on `SUSE` or `openSUSE Leap`, `15.x` is the only supported release for both, right?

Reply:
I have tested ceph-ansible (using the various tweaks, see the lot of other PRs) successfully on both SLES 15.x and openSUSE Leap 15.x.

I am not sure if anyone tested ceph-ansible on SLES12, I have this on my list. Once I am sure this works, I would adapt this condition once again.

We can of course wait until my tests are done, but that might take a while.

Reply:
> I am not sure if anyone tested ceph-ansible on SLES12, I have this on my list.

yeh, I think that's fine merging this as it is then.

Reply:
For the record:

> I am not sure if anyone tested ceph-ansible on SLES12, I have this on my list. Once I am sure this works, I would adapt this condition once again.

SLES12 SP1 fails to install chrony, as no chrony package is available, so it would need some more modifications for that to work. Not sure if SLES12 SP4 already has that package available, I will check and prepare something working in another pull request.